app/styles/utils/_layoututils.scss

```diff
@@ -14,6 +15,8 @@ $padding-xlarge: 26px;
 $padding-xxlarge: 36px;
 $max-container-width: 1280px;
 
+$header-height: 70px;
+$header-mobile-height: 50px;
```

Comment:
Although we probably don't want to do it in this PR, we may want to consider migrating to [CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/--*) (a.k.a. CSS variables), which have the advantage of being able to observe media queries, since they are actually built-in to CSS itself. We currently make use of this in the sheltertech.org site, and it's actually very nice, since we can reuse the same variables between desktop and mobile views: https://github.com/ShelterTechSF/sheltertech.org/blob/714aefc95629b05c10dc207c007ad9c2f59d5804/src/stylesheets/global.css#L74-L86

Reply:
That sounds great. I will keep that in mind next time I want to create/edit a CSS var